src/Lecture3.hs

```diff
@@ -93,7 +108,10 @@ weekday to the second.
 >>> daysTo Friday Wednesday
 5
 -}
-daysTo = error "TODO"
+daysTo :: Weekday -> Weekday -> Int
+daysTo x y 
+  | x == y = 0
+  | otherwise = 1 + daysTo (csucc x) y
```

Comment:
That's a correct implementation! 👍🏻 
You can make it more efficient and avoid recursion by using math operations 🔢

Reply:
I see, let me update the solution.